src/applications/personalization/view-dependents/components/ViewDependentsFormStates/BuildFormlett.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ViewDependentsChildForm from './ViewDependentsChildForm.jsx';
+import ViewDependentsSpouseForm from './ViewDependentsSpouseForm.jsx';
+
+function buildFormlett(relationship) {
+  let formlett = '';
+  switch (relationship) {
+    case 'Child':
+      formlett = <ViewDependentsChildForm />;
+      break;
+    case 'Spouse':
+      formlett = <ViewDependentsSpouseForm />;
```

Comment:
Possibly worth exploring in a future refactor / iteration, but it might be cool to use some sort of factory to generate more flexible schemas and child components instead of declaring them statically per dependent type.